common/map.go

```diff
@@ -0,0 +1,168 @@
+package common
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+)
+
+type GenericMap map[string]any
```

Comment:
Can you name this `StringMap`? Just because "generic" has a specific meaning in Go.